CONTRIBUTING.md

```diff
@@ -41,75 +25,97 @@ process documented here.
 
 All contributors must agree to the project [code of conduct](CODE_OF_CONDUCT.md).
 
-## Golang Coding Style
+## Pull requests
+
+All the repositories accept contributions by a GitHub Pull request (PR).
+
+## Git basic setup
+
+To get started, complete the prerequisites below.  
+
+### Prerequisites
+
+- Review [Contributor roles](#Contributor-roles) for special configuration
+
+- [Set up Git](https://help.github.com/en/github/getting-started-with-github/set-up-git). *Note:* The email address you specify must match your primary  GitHub email address.
```

Comment:
I'm not sure if the 'must match your primary email address' is correct. The email you sign off by must match the email that gets placed into the commit I think, otherwise the CI complains - but, I'm not sure it has to be the primary github email address. Do you know specifics or details @jodh-intel ?

Reply:
I agree with @grahamwhaley. I'll use "The email address you specify must match the email address that you use to sign-off commits."  This is more accurate.